cloudera/README.md

```diff
@@ -81,6 +81,44 @@ For containerized environments, see the [Autodiscovery Integration Templates][3]
 <!-- xxz tab xxx -->
 <!-- xxz tabs xxx -->
 
+#### Clusters Discovery
+
+You can configure how your clusters are discovered with the `clusters` parameter.
+
+`limit`
+: Maximum number of items to be autodiscovered.  
+**Default value**: `None` (all clusters will be processed)
+
+`include`
+: Mapping of regular expression keys and component config values to autodiscover.  
+**Default value**: empty map
+
+`exclude`
+: List of regular expressions with the patterns of components to exclude from autodiscovery.  
+**Default value**: empty list
+
```

Comment:
I think you are also missing `interval` parameter here?

Reply:
Done